purchase_analytic/i18n/purchase_analytic.pot

```diff
@@ -0,0 +1,34 @@
+# Translation of Odoo Server.
```

Comment:
There are missing commits present in 16.0, mainly related to translations, e.g. de8e27fa84267343190adc2874042efe9593cddf. I'd suggest retrieving the history again and then cherrypicking the migration commits.